lib/common.ts

```diff
@@ -119,8 +119,9 @@ function injectSnapshot(file: string) {
       // C:\
       file = file.slice(0, -1);
     }
-
-    return `${file[0]}:\\snapshot${file.slice(2)}`;
+    // by convention, on windows we use C:\\snapshot
+    const pathInSnapshot = `C:\\snapshot${file.slice(2)}`;
+    return pathInSnapshot;
```

Comment:
return directly here?

lg2m in principle.

Reply:
Yes, , I had a console.log at some point while developing, hence I store the intermediate result in a variable. 

Btw, I am struggling to find a way to create a non-regression test for this issue. as I am not sure we can easily create an alternative drive on the CI/CD Windows build the image.

Reply:
You can mount a VHD.

Reply:
I found that `subst H: C:\MyFolder` will work.